### DISCUSSION TITLE
Pull Shark - Change `inaccessible` word

### DISCUSSION THREAD

**Discussion by kblok** [community/community#18265] [Other Feature Feedback, Questions, & Ideas]
I think we can use another word instead inaccessible here. Private? Deleted? Something more... human.

<img width="361" alt="image" src="https://user-images.githubusercontent.com/2198466/173063756-087b8c7e-3656-46bd-8674-6c31d8f8c734.png">

**Comment by byronbytes** [comment]
That's most likely a placeholder text, but I do believe that "Private Repo" would be a better fit, or something that indicates it as private / deleted.

**Comment by prrao87** [comment]
Yes, please change this! It's quite annoying to see placeholder text like "inaccessible" when it's clear that it's from a private repo.

**Comment by Beka0n** [comment]
"Private Repo" will look better.

**Comment by sam0hack** [comment]
I'm sure they would change it soon, right now it is in beta So new changes may come in the next release. instead of _inaccessible_ Private repo is a good idea 💡
